BringgSDK.js

```diff
@@ -950,7 +969,19 @@ var BringgSDK = (function () {
     });
   }
 
-  function createShareForOrderViaRest(orderUuid, customerAccessToken) {
+   function getOrderViaRestByAccessToken(shareUuid, accessToken) {
+    log('Getting order via REST with share uuid: ' + shareUuid);
+    $.getJSON(getRealTimeEndPoint() + 'watch/shared/' + shareUuid + '?access_token=' + accessToken, function (result) {
+      log('Rest order update: ' + JSON.stringify(result));
+      if (result.success && result.order_update) {
+        module._onOrderUpdate(result.order_update);
+      }
+    }).error(function (jqXHR, textStatus, errorThrown) {
+
```

Comment:
this is out of the scope of this pr, but now that i see this, we should probably consider delegating the error handling here (and in the other rest api functions) to the callbacks instead of swallowing it.